sdk/containerregistry/azure-containers-containerregistry/src/test/java/com/azure/containers/containerregistry/ContainerRepositoryUpdateTests.java

```diff
@@ -7,220 +7,226 @@
-import org.junit.jupiter.api.parallel.ExecutionMode;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.MethodSource;
-
-import java.util.Arrays;
+import com.azure.core.http.rest.PagedIterable;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.Context;
+import org.junit.jupiter.api.Test;
+
 import java.util.List;
 import java.util.stream.Collectors;
 
-import static com.azure.containers.containerregistry.TestUtils.DISPLAY_NAME_WITH_ARGUMENTS;
-import static com.azure.containers.containerregistry.TestUtils.LIBRARY_BUSYBOX_NAME;
+import static com.azure.containers.containerregistry.TestUtils.DIGEST_UNKNOWN;
+import static com.azure.containers.containerregistry.TestUtils.HELLO_WORLD_SEATTLE_REPOSITORY_NAME;
 import static com.azure.containers.containerregistry.TestUtils.TAG_TO_DELETE;
 import static com.azure.containers.containerregistry.TestUtils.TAG_TO_UPDATE;
+import static com.azure.containers.containerregistry.TestUtils.TAG_UNKNOWN;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
-@Execution(ExecutionMode.SAME_THREAD)
 public class ContainerRepositoryUpdateTests extends ContainerRegistryClientsTestBase {
+    private String recordFileName;
+    private static final String PARENT_FILENAME = "ContainerRepositoryAsyncClientUpdateIntegrationTests.";
+    private ContainerRepositoryClient client;
 
-    private ContainerRepositoryClient getContainerRepositoryClient(HttpClient httpClient) {
-        return getContainerRepositoryBuilder(LIBRARY_BUSYBOX_NAME, httpClient).buildClient();
+    private ContainerRepositoryClient getContainerRepositoryClient() {
+        return getContainerRepositoryBuilder(HELLO_WORLD_SEATTLE_REPOSITORY_NAME, new LocalHttpClient(recordFileName)).buildClient();
     }
 
-    private ContainerRegistryClient getContainerRegistryClient(HttpClient httpClient) {
-        return getContainerRegistryBuilder(httpClient).buildClient();
+    private ContainerRepositoryClient getUnknownContainerRepositoryClient() {
+        return getContainerRepositoryBuilder(TAG_UNKNOWN, new LocalHttpClient(recordFileName)).buildClient();
     }
 
-    private boolean resetTag;
-    private boolean resetRepository;
-    private String digest;
-
-    @BeforeEach
-    public void setup() {
-        resetTag = false;
-        digest = null;
-        importImage(LIBRARY_BUSYBOX_NAME, Arrays.asList(TAG_TO_UPDATE, TAG_TO_DELETE));
+    private ContainerRegistryClient getContainerRegistryClient() {
+        return getContainerRegistryBuilder(new LocalHttpClient(recordFileName)).buildClient();
     }
 
-    @AfterEach
-    public void cleanup() {
-        ContainerRepositoryClient client = getContainerRepositoryClient(new NettyAsyncHttpClientBuilder().build());
+    @Test
+    public void setRepositoryProperties() {
+        recordFileName = PARENT_FILENAME + "setRepositoryPropertiesWithResponse[1].json";
```

Comment:
Using these one off file recording names gives me some concern from a maintainability perspective. This goes against the normal pattern where a test recording uses the file name `<class>.<method>[<method iteration>].json` and could cause unexpected test failures when changes are made to azure-core-test